src/rendering/screenspacerenderable.cpp

```diff
@@ -146,10 +146,11 @@ namespace {
         openspace::properties::Property::Visibility::NoviceUser
     };
 
-    constexpr openspace::properties::Property::PropertyInfo GammaInfo = {
-        "Gamma",
+    constexpr openspace::properties::Property::PropertyInfo GammaOffsetInfo = {
+        "GammaOffset",
         "Gamma Correction",
```

Comment:
Should this name maybe be "Gamma Correction Offset"? 

To make it clearer that the value is now an offset